src/App.jsx

```diff
@@ -39,28 +39,28 @@ class App extends React.Component {
                         {/* <Route exact path="/verify" key="Verify" render={ props => <CustomAuthenticator onSignIn={this.signInHandler}/> } /> */}
 
-                        <Route exact path="/about" key="Who are we" component={About}/>
+                        <Route exact path="/over-ons" key="Who are we" component={About}/>
                         <Route exact path="/contact" key="Contact us" component={Contact}/>
-                        <Route exact path="/privacy" key="Privacy Statement" component={Privacy}/>
-                        <Route exact path="/faq" key="Frequently Asked Questions" component={FAQ}/>
-                        <Route exact path="/terms" key="Terms of Use" component={Terms}/>
+                        <Route exact path="/privacyverklaring" key="Privacy Statement" component={Privacy}/>
+                        <Route exact path="/veelgestelde-vragen" key="Frequently Asked Questions" component={FAQ}/>
+                        <Route exact path="/gebruiksvoorwaarden" key="Terms of Use" component={Terms}/>
 
-                        <Route exact path="/workspace/" key="Workspace" component={Workspace}/>
+                        <Route exact path="/actie/" key="Workspace" component={Workspace}/>
 
-                        {/* <Route exact path="/workspace/:initiativeId" key="WorkspacePage" component={Workspace}/>
-                        <Route exact path="/workspace/:initiativeId/login" key="WorkspaceLogin" render={ props => <Workspace {...props} signInHandler={this.signInHandler}/> } /> */}
+                        {/* <Route exact path="/actie/:initiativeName" key="WorkspacePage" component={Workspace}/>
+                        <Route exact path="/actie/:initiativeName/inloggen" key="WorkspaceLogin" render={ props => <Workspace {...props} signInHandler={this.signInHandler}/> } /> */}
                         
-                        <Route exact path="/workspace/login" key="WorkspaceLogin" render={props => <Workspace {...props} signInHandler={this.signInHandler}/>}/>
-                        <Route exact path="/workspace/add-find-playground" key="WorkspaceAddFindPlayground" component={Workspace}/>
+                        <Route exact path="/actie/inloggen" key="WorkspaceLogin" render={props => <Workspace {...props} signInHandler={this.signInHandler}/>}/>
+                        <Route exact path="/actie/starten" key="WorkspaceAddFindPlayground" component={Workspace}/>
 
-                        <Route exact path="/workspace/:initiativeId/join" key="WorkspaceJoin" component={WorkspaceJoin}/>
-                        <Route exact path="/workspace/:initiativeId/login" key="WorkspaceJoin" render={props => <Workspace {...props} signInHandler={this.signInHandler}/>}/>
+                        <Route exact path="/actie/:initiativeName/join" key="WorkspaceJoin" component={WorkspaceJoin}/>
```

Comment:
join = aansluiten

Reply:
But if it needs for SEO purposes so ok, I'll rename it.

Reply:
Done